assemblies/assembly_openshift-operator/main.adoc

```diff
@@ -0,0 +1,51 @@
+[id='assembly_openshift-operator']
+
+include::_artifacts/document-attributes.adoc[]
```

Comment:
Not in this PR, but the "Providing the LDAP role mapping file" section has a few things:

* " from the file. Run the following command:"  > "from the file by running the following command:" Right?
* "every deployment config ... . The following deployment configs ..."  >  "deployment configuration ... deployment configurations"
* Also, below the same section in my above bullet, the example snippet is missing. Goes from "can be affect in this environment: ..." to just "Where my-app ..."
* For every deployment configuration, run the command:"  >  "... run the following command:"